packages/cli/src/commands/generate/service/service.js

```diff
@@ -52,7 +52,8 @@ export const scenarioFieldValue = (field) => {
     case 'Boolean':
       return true
     case 'DateTime':
-      return new Date()
+      // use a static date so that snapshots don't change
+      return new Date('2022-09-01T00:21:58.876Z')
```

Comment:
not loving using a static date, but couldn't find a way to make the snapshots work for the other cases. thoughts?

Reply:
It's random values for a few other fields, like `Int`. Why can those be random, but not the date? I'll try to look into it myself, but if you already know, please tell me 🙂

Reply:
Math.random is mocked. That's why. I'll see if I can mock new Date() too

Reply:
Done 🙂